### ISSUE TITLE
Unstable connection for radsec.openroaming.goog

### ISSUE THREAD

**Issue by hgot07** [radsecproxy/radsecproxy#131]
I'm not sure whether the cause is in radsecproxy or on Google side.
We are experiencing unstable connection for radsec.openroaming.goog using a profile generated by Pixel's built-in OSU. 

The symptom is as follows.
(tested with radsecproxy 1.10.0, 1.9.3, and 1.9.2, all affected)

1. Just after invoking radsecproxy, the RadSec + DPD works well for some time.
2. After a few minutes of silence w/o request, the RADIUS packets seem being blocked. Access-Request packets won't receive any reply.
3. After some tens of minutes, the RadSec + DPD starts working well.
4. (repeat from 2)

Log file and a summary are attached. The requests between line 672-1181 are affected.
(Please ignore some lines for Samsung, etc.)

[messages-0222.txt](https://github.com/radsecproxy/radsecproxy/files/12078417/messages-0222.txt)
[radsecproxy-summary.txt](https://github.com/radsecproxy/radsecproxy/files/12078420/radsecproxy-summary.txt)

**Comment by fmauchle** [comment]
Could you provide your dynamic `realm` config and associated `server` config(s) too please?

**Comment by hgot07** [comment]
The naptr lookup script can be found at: https://github.com/wireless-broadband-alliance/openroaming-config/blob/main/naptr-openroaming.sh 

server OR-dynamic {
        type tls
        tls OR-client
        secret radsec
        certificateNameCheck off
        dynamicLookupCommand /etc/radsecproxy/naptr-openroaming.sh
}

realm /@.+\..+$/ {
        server OR-dynamic
        accountingServer OR-dynamic
        accountingResponse on
}

Google support looked into their logs and wrote 
> the missing requests do not appear to be reaching us

**Comment by fmauchle** [comment]
Ok, I'm suspecting an interaction when both `server` and `accountingServer` are dynamic.

As a short-term mitigation/test could you try to eiter
- remove the `accountingServer` config or (if this is not an option)
- enable `statusServer`

note the latter will likely keep the connection open forever.

**Comment by hgot07** [comment]
Thanks for your comments.
I've inserted the following realm block.
**The symptom seems to have gone without `accountingServer`.**

```
realm openroaming.goog {
        server OR-dynamic
        accountingResponse on
}
```

**The issue has come back after I added  `accountingServer` to this block.**

I'm using exact realm match for this test case since I didn't want to change the behaviors for other IdPs.

Please note that such a per-IdP definition isn't useful in OpenRoaming as there are so many IdPs.

I haven't tested the second option (statusServer). 
It won't be practical as it requires per-IdP server definitions.

**Comment by hgot07** [comment]
I've tested `statusServer on` option today, but the issue is still there.

**Comment by hgot07** [comment]
It seems the fix in maint-1.10 is effective. Once the next release is out and I confirm the functionality, this issue report will be closed.

**Comment by hgot07** [comment]
The latest version 1.10.1 has been working well in the last few days. The issue is probably gone. Thank you very much for the fix.
